packages/jest-runner/worker.ts

```diff
@@ -215,15 +232,91 @@
 		});
 	}
 
-	// noinspection JSUnusedLocalSymbols
 	private async runHooks(
 		hookType: string,
-		block: Circus.DescribeBlock | Circus.TestEntry,
+		block: Circus.DescribeBlock | Circus.TestEntry, // TODO: kill testEntry
```

Comment:
Remove `TestEntry`as described in the comment?

Reply:
Done!